mito-ai/src/Extensions/AiChat/ChatTaskpane.tsx

```diff
@@ -69,6 +70,31 @@ import LoadingCircle from '../../components/LoadingCircle';
 import { checkForBlacklistedWords } from '../../utils/blacklistedWords';
 import DropdownMenu from '../../components/DropdownMenu';
 
+// Suggestion items for chat prompt boxes
+interface SuggestionOption {
+    display: string;
+    prompt: string;
+}
+
+const DEFAULT_SUGGESTION_OPTIONS: SuggestionOption[] = [
+    {
+        display: "Plot Meta Acquisitions Impact",
+        prompt: "Build an annotated graph of how Meta's acquisitions of Instagram, Whatsapp, and Giphy affected the Meta stock price. Use the data from https://raw.githubusercontent.com/mito-ds/mito/refs/heads/dev/jupyterhub/meta_stock_prices.csv"
```

Comment:
There's a visual bug with these long URLs (not introduced in this PR), but let's fix it up now since it's triggered with these prompts. 
<img width="397" alt="Screenshot 2025-04-25 at 5 23 25 PM" src="https://github.com/user-attachments/assets/2d5d396d-eeb6-473d-b17f-fef318380dac" />

Reply:
When the big urls exist, I am able to scroll the taskpane to the right and its looks like this. 

Instead, we should make the urls wrap or just cut them off with `...`